src/plugins/expression_reveal_image/common/lib/elastic_logo.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const elasticLogo =
```

Comment:
nit: not related to your changes but svg convert to base64 looks like a good candidate for refactoring. Can we discuss it with code owner?

Reply:
+ I see that some teams use svg imports. It means that we already have configured loader for svg file type

![image](https://user-images.githubusercontent.com/20072247/122221554-0cdd4a00-ceba-11eb-8a43-3a0896388402.png)